tests/reftests/lint.test

```diff
@@ -401,6 +401,19 @@ ${BASEDIR}/lint.opam: Warnings.
            warning 47: Synopsis should start with a capital and not end with a dot
 ### <lint.opam>
 opam-version: "2.0"
+description: "a words."
+authors: "the testing team"
+homepage: "egapemoh"
+maintainer: "maint@tain.er"
+license: "ISC"
+dev-repo: "hg+https://to@li.nt"
+bug-reports: "https://nobug"
+### opam lint ./lint.opam
+${BASEDIR}/lint.opam: Errors.
+             error 57: Synopsis must not be empty
+# Return code 1 #
+### <lint.opam>
+opam-version: "2.0"
```

Comment:
This check and the one below need to go on `E57` section line 624.

Reply:
also, i forgot to change W47 comment line with the new description (line 388), can you update it and E57 one too (line 624) ?